### PR TITLE
List bug fix

### PR DESCRIPTION
**Pull request by AllardQuek**
- List now returns message saying no students to display if there are no students stored
- Improve test quality

### CODE REVIEW

File: src/test/java/seedu/programmer/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public void setUp() {
 
     @Test
     public void execute_invalidCommandFormat_throwsParseException() {
-        String invalidCommand = "uicfhmowqewca";
+        String invalidCommand = "inavlid command";
```
Comment (reddevil1313) [inline src/test/java/seedu/programmer/logic/LogicManagerTest.java:55]: Is this a typo here or intentional?

Comment (AllardQuek) [reply src/test/java/seedu/programmer/logic/LogicManagerTest.java:55]: Intentional. "invalid command" is still an invalid command, and provides more info to the developer reading

Comment (reddevil1313) [reply src/test/java/seedu/programmer/logic/LogicManagerTest.java:55]: okay, makes sense!

**Review by reddevil1313 — APPROVED** [review]
LGTM

**Pull request merged**